private/projects/pdk-monorepo-project.ts

```diff
@@ -205,8 +196,16 @@ export class PDKMonorepoProject extends NxMonorepoProject {
    */
   synth() {
     this.subProjects.forEach((subProject) => {
+      subProject.packageTask.prependExec(
+        `pnpx ts-node ${path.relative(
+          subProject.outdir,
+          this.outdir
+        )}/scripts/jsii-pacmak-hack.ts`
+      );
```

Comment:
Is the jsii-packman PR not published yet? Once that is this will get removed?

Can you add some comments regarding the clean up for this, and an issue to remove this once that PR is published.